frontend/src/pages/EditProfilePage/index.jsx

```diff
@@ -94,11 +104,10 @@ const EditProfilePage = ({ uid }) => {
         />
         <Text>Classes</Text>
         <AppDropDown
-          onSelect={addClass}
+          onSelect={dropDownHandler}
```

Comment:
Where is onSelect used inside the AppDropDown?

Reply:
yes, I forgot to moved the changes this branch, will fix